### PR TITLE
Limit broadcast sending to contacts and urns for the omnibox on the i…

### PR DESCRIPTION
**Pull request by norkans7**
…nbox page

### CODE REVIEW

**Comment by nicpottier** [comment]
This looks like this is changing all omnibox instances no? How does this change the "Send Message" action in a flow?

**Comment by norkans7** [comment]
It is only changing the one on the inbox page and the contact read/list pages 

the flow editor uses https://github.com/nyaruka/rapidpro/blob/master/static/coffee/flows/widgets.coffee#L477

**Comment by norkans7** [comment]
I also pushed this to staging

**Comment by nicpottier** [comment]
The help text still says "enter contacts or groups"

File: static/js/omnibox.js [hunk 1]
```diff
@@ -28,6 +28,8 @@ function omnibox(ele, types, options) {
     var placeholder = null;
     if (types == 'g'){
         placeholder = gettext("Enter one or more contact groups");
+    } else if (types == 'cu'){
+        placeholder = gettext("Recipients, enter contacts or numbers");
```
Comment (rowanseymour) [inline static/js/omnibox.js]: "or phone numbers"

File: static/js/omnibox.js [hunk 2]
```diff
@@ -97,17 +99,20 @@ function omnibox(ele, types, options) {
...
+    if (types === undefined) {
+        types = 'cg';
+    }
+    omnibox($(".omni_widget"), types);
 }
 
 function initializeOmnibox(initial) {
     var options = {
-        placeholder: gettext("Recipients, enter contacts or groups"),
+        placeholder: gettext("Recipients, enter contacts or numbers"),
```
Comment (rowanseymour) [inline static/js/omnibox.js]: as above

**Pull request merged**